packages/react-examples/README.md

```diff
@@ -1,47 +1,77 @@
+
+Since we mostly didn't end up splitting packages in v8, and went with a different approach for v9, the only package where the circular dep issue is relevant now is `@fluentui/react-focus`. Otherwise, the examples could theoretically be moved back into component packages.
+
+## Local development
+
+### Storybook
+
+To start Storybook for `@fluentui/react`, run:
+
+```
+yarn workspace @fluentui/react start
+```
+
+The same command works for any other package with examples located under this folder; just be sure to use the _full package name_.
+
+### Legacy demo app
 
-Each folder under `src` is the unscoped name of a package, containing examples for that package. For example, the examples for `@fluentui/react` are under `src/react`.
+To start the legacy demo app for `@fluentui/react`, run:
+
+```
+yarn workspace @fluentui/react start:legacy
+```
 
-The examples under `react` (plus `react-focus` and `react-cards`) are used on the [public documentation site](https://developer.microsoft.com/fluentui) via `apps/public-docsite`, and [aka.ms/fluentdemo](https://aka.ms/fluentdemo) via `apps/public-docsite-resources`.
+This also works with `@fluentui/react-charting` and `@fluentui/react-experiments`.
 
-The other packages' examples are only used for local development and on the [PR deploy site](https://fluentuipr.z22.web.core.windows.net/heads/master/).
+## Architecture
 
 ### Structure of package documentation folders
 
-1. Folder name reflects a package name omitting the npm scope.
+> `src/azure-themes` doesn't follow this convention.
+
+Within each package folder, usually each sub-folder contains the examples and documentation site content for a particular component (or occasionally a category of components, such as `Pickers`).
 
-For example `src/react` contains stories and documentation site content for `@fluentui/react`
+For example, `src/react/Panel` contains examples and docs for `@fluentui/react`'s `Panel` component.
 
-2. Sub folders are named by sub-component that is part of the reflected package
+`src/react-charting` and `src/react-experiments` also have a `demo` sub-folder with legacy demo app configuration.
 
-For example `src/react/Panel` contains stories and documentation site content for `@fluentui/react` package `Panel` component
+### File naming conventions
 
-**Subfolder contains following types files with enforced naming:**
+> `src/azure-themes` is not subject to these constraints.
 
-1. example/story files: `<ComponentName>.<ExampleName>.Example.tsx`
+1. Example/story files: `<ComponentName>.<ExampleName>.Example.tsx`
 
-   - file name needs to match identifier name being exported from the file.
+   - By convention, the exported component's name should match the file name.
+   - For example: `Panel.Basic.Example.tsx` should export a function component or class component called `PanelBasicExample`.
 
-     - > Note that violating this will not render your stories!
```

Comment:
I tested this and confirmed the story export naming is just a convention, not a hard requirement